core/src/main/scala/org/scalatra/servlet/RichResponse.scala

```diff
@@ -21,28 +18,10 @@ case class RichResponse(res: HttpServletResponse) {
     res.setStatus(status)
   }
 
-  object headers extends Map[String, String] {
-
-    def get(key: String): Option[String] =
-      res.getHeaders(key) match {
-        case xs if xs.isEmpty => None
-        case xs => Some(xs.asScala mkString ",")
-      }
-
-    def iterator: Iterator[(String, String)] =
-      for (name <- res.getHeaderNames.asScala.iterator)
-        yield (name, res.getHeaders(name).asScala mkString ", ")
-
-    def +=(kv: (String, String)): this.type = {
-      res.setHeader(kv._1, kv._2)
-      this
+  object headers {
+    def update(name: String, value: String): Unit = {
```

Comment:
`set` might be better? and don't we need other methods like `names`, `get` and `getOrElse`?

Reply:
Since there are many places where update syntax is used, update method has been added.
However, from the viewpoint of clarity, I add set method together.

The get method is not prepared because there is no method to get the header in the response of the servlet